ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/validation/validation.tsx

```diff
@@ -16,7 +16,33 @@
 import { InvalidSearchFormMessage } from '../../../component/announcement/CommonMessages'
 const announcement = require('../../../component/announcement/index.jsx')
 
-export function validate(errors: any) {
+export function validate(filters: any, errors: any) {
+  for (let i = 0; i < filters.length; i++) {
+    const filter = filters[i]
+    const geometry =
+      filter.geojson && filter.geojson.geometry ? filter.geojson.geometry : null
```

Comment:
:pencil2: can remove the ternary and just do `filter.geojson && filter.geojson.geometry`